CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+Contributing
+============
+
+Make some changes, open a PR, easy-peasy. I like to manually apply PRs, so don't be
+surprised if I close your PR while pushing your authored commit directly to main.
+
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the work by you shall be licensed as in [LICENSE](LICENSE),
+without any additional terms or conditions.
```

Comment:
I don't intend for any licensing changes so I think this should be reworded. Something like "All contributions are subject to the LICENSE." or equivalent.

Reply:
Yeah, that's good for me.